docs/conf.py

```diff
@@ -48,7 +48,7 @@
 
 # General information about the project.
 project = u'Pillow (PIL Fork)'
-copyright = u'1995-2016, Fredrik Lundh and Contributors, Alex Clark and Contributors'
+copyright = u'1995-2017, Fredrik Lundh and Contributors, Alex Clark and Contributors'
```

Comment:
Here's the PIL copyright in the README from when it was forked: https://github.com/python-pillow/Pillow/blob/9a640e3157150e35f17354517e8c6304fc428060/README#L98-L99